src/main/java/core/basesyntax/service/FileReader.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileReader {
+    List<String> fruitsList(String file);
```

Comment:
interface is called FileReader but method is fruitsList, just doesn't make sense